src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/config/LocalStackConfig.kt

```diff
@@ -17,29 +19,55 @@ import org.springframework.context.annotation.Primary
 @Configuration
 class LocalStackConfig {
 
+  companion object {
+    val logger = LoggerFactory.getLogger(this::class.java)
+  }
+
   @Bean("awsSqsClient")
   @Primary
   @ConditionalOnProperty(name = ["sqs.provider"], havingValue = "localstack")
-  fun awsSqsClientLocalStack(
-    @Value("\${sqs.endpoint.url}") serviceEndpoint: String,
-    @Value("\${sqs.endpoint.region}") region: String
-  ): AmazonSQSAsync =
+  fun sqsClient(sqsConfigProperties: SqsConfigProperties, dlqSqsClient: AmazonSQS): AmazonSQSAsync =
+    amazonSQSAsync(sqsConfigProperties.localstackUrl, sqsConfigProperties.region)
+      .also { sqsClient -> createMainQueue(sqsClient, dlqSqsClient, sqsConfigProperties) }
+      .also { logger.info("Created sqs client for queue ${sqsConfigProperties.queueName}") }
```

Comment:
BTW I think logging the name means you could use it as a parameter to an external script to add messages to a queue. I think this is "good enough"